### DISCUSSION TITLE
Stream not being saved to disk(?)

### DISCUSSION THREAD

**Discussion by hsg120** [bluenviron/mediamtx#3296] [Questions]
### Question

I'm certain I'm doing something wrong; I'm just at a loss as to what it is. 

Running v1.5.1 on Arch.

My camera (Wyze cam) is serving an RTSP stream, and if I put the address into VLC I can see the stream, no problem.

But, when I put the address into the mediamtx.yml config file and start the server there's no saved streams. The log doesn't show any errors. It just says listening ports are open. Recording is enabled in the config file, and I have the RTSP address entered as my "source".

I'm probably doing something dumb in the config file, which is attached. 
[mediamtx.txt](https://github.com/bluenviron/mediamtx/files/15123646/mediamtx.txt)

Any help would be so so appreciated!!
[mediamtx.log](https://github.com/bluenviron/mediamtx/files/15123663/mediamtx.log)

**Comment by aler9** [comment]
Hello, as written in the README, in order to pull a stream from a RTSP server, you have to create a dedicated path configuration:

```yml
paths:
  my_camera:
    source: rtsp://192.168.0.219:8554/unicast
```

[a comment by an automated account is not shown]
